### PR TITLE
Form Recognizer switch to AAD auth

### PR DESCRIPTION
**Pull request by samvaity**
This PR adds
- Form Recognizer switch to AAD 
- Use static resource for FR Live tests
- Workaround in tests for DatatTable.BoundingBox returned as a null issue.
- Minor test improvements.

### CODE REVIEW

File: sdk/formrecognizer/test-resources.json [hunk 1]
```diff
@@ -11,7 +11,7 @@
         },
         "location": {
             "type": "string",
-            "defaultValue": "westus2",
+            "defaultValue": "centraluseuap",
```
Comment (maririos) [inline sdk/formrecognizer/test-resources.json:14]: do u want to change to canada central? or leave as is

Comment (maririos) [reply sdk/formrecognizer/test-resources.json:14]: hmmm I guess it doesn't matter as there is a static resource

**Review by maririos — APPROVED** [review]

**Review by kristapratico — APPROVED** [review]

File: sdk/formrecognizer/tests.yml [hunk 2]
```diff
@@ -8,5 +8,21 @@ jobs:
         - name: azure-ai-formrecognizer
           groupId: com.azure
           safeName: azureaiformrecognizer
+      # resource creation is disabled because we are trying to test the stable endpoint.	
+      # Will enable it when service rolls out AAD to all region.	
+      DisableAzureResourceCreation: true
```
Comment (kristapratico) [inline sdk/formrecognizer/tests.yml:13]: Didn't know about this, cool

**Review by mssfang — APPROVED** [review]

File: sdk/formrecognizer/tests.yml (hunk 2 above)
Comment (samvaity) [reply sdk/formrecognizer/tests.yml]: Hehe...Thanks to @mssfang and TA to have done this before 🚆 .

**Pull request merged**